geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/operations/GetRequestOperationHandler.java

```diff
@@ -53,10 +52,10 @@
           ProtobufUtilities.createEncodedValue(serializationService, resultValue);
       return Success.of(RegionAPI.GetResponse.newBuilder().setResult(encodedValue).build());
     } catch (UnsupportedEncodingTypeException ex) {
-      return Failure.of(
-          ClientProtocol.ErrorResponse.newBuilder().setMessage("Encoding not supported.").build());
+      return Failure
```

Comment:
We're not entirely consistent with when we log errors and when we don't. Probably these should be logged like the others?